wordpress-mobile/WordPress-Android
# Issue/14004 menu item order

Fixes #14004 

This PR changes the More menu on the ReaderPost card
- The menu item order has changed
- The block site/report this post test/icon have changed color to red
- There is an empty row between share and the block/report actions

|current |light|dark
|---|---|--|
|![current](https://user-images.githubusercontent.com/506707/109880558-13b2ba80-7c45-11eb-9ecc-c7d58b8dd5f3.jpg)|![light](https://user-images.githubusercontent.com/506707/109880798-5f656400-7c45-11eb-977e-a51b5099c7ad.png)|![dark](https://user-images.githubusercontent.com/506707/109880595-21684000-7c45-11eb-8b36-0c4083650cea.png)

**To test:**
Scenario One
- Launch the app
- Navigate to discover
- Tap the more menu on a card.
- Note that following:
-- The item order has changed
-- There is a space before the block this site and/or report this site
-------------------
Scenario Two
- Launch the app
- Navigate to following
- Tap the more menu on a card.
- Note that following:
-- The item order has changed
-- There is a space before the block this site and/or report this site
PR submission checklist:
